Jenkinsfile

```diff
@@ -126,11 +129,13 @@ pipeline {
                         }
                         echo "Using version ${Globals.version}"
                         if (env.BRANCH_NAME == 'main') {
-                            Globals.imageTag = "${Globals.IMAGE_NAME}:latest"
+                            Globals.imageTagIntern = "${Globals.IMAGE_NAME_INTERN}:latest"
+                            Globals.imageTagPublic = "${Globals.IMAGE_NAME_PUBLIC}:latest"
```

Comment:
I don't really have a full view on the deployment strategy but I have the impression that this might cause some issues with caching and the ability to control which image gets deployed. It's good from the point of view that amount storage required will not grow with time though. That said, this could be solved in a different PR.

Reply:
For the moment I'm mainly using the stage "pushing the artifacts" to run them on the VM and not the "deploy" stage, which fails.

Reply:
That's fine. I was not referring to the deploy stage in the pipeline. At the moment, even if you deploy manually, you will need to tell the cache layer somehow that there is a new image tagged `latest` such that it will download the new image.